data/console/test_java.sh

```diff
@@ -219,11 +236,26 @@ for java_version in $(cat $LIST_ALL_INSTALLED_VERSIONS); do
         echo "Error: java alternative not found for $java_version"
         exit 3
     fi
-    # Test if there's an alternativ for javac, and if yes, set it as the current used one
+    # Test if there's an alternative for javac, and if yes, set it as the current used one
     if grep $dot_version $LIST_ALL_JAVAC_ALTERNATIVES > /dev/null; then
         javac=$(grep $dot_version $LIST_ALL_JAVAC_ALTERNATIVES)
         update-alternatives --set javac $javac
     else
+        if echo $java_version | grep ibm > /dev/null; then
+            if ! rpm -qa | grep java | grep devel | grep $dot_version > /dev/null; then
+                echo "Warning: java compiler alternative not found for $java_version"
+                echo "Reason : devel pkg is not installed, thus it is normal there is not javac"
+                if zypper products -i | grep sle-sdk > /dev/null; then
+                    echo "Status : Error! The devel pkg should be installed. Check your repos!"
+                    exit 8
```

Comment:
Is there any guideline for openQA return codes? I was using mine, which kind of pointless because they are known only to me, so I will change all the (fail) return codes to 1, which is the default that everybody expects in such a case.